SteamVR-WebKit/JsInterop/Notifications.cs

```diff
@@ -22,48 +22,39 @@ struct NotificationIcon
             public Bitmap Bitmap;
             public int glTextureId;
         }
-
-        private CVRNotifications _notificationsInstance;
-        private Overlay _overlay;
+        
+        private ulong _handle = 0;
         private static Dictionary<string, NotificationIcon> _icons = new Dictionary<string, NotificationIcon>();
 
-        public Notifications(Overlay overlay)
+        public Notifications(ulong overlayHandle)
```

Comment:
Please stick to passing the actual Overlay.cs instances to these objects rather than just the handle, it's more useful in the long run.